### PR TITLE
appended a zero to group code

### PR DESCRIPTION
**Pull request by shejasabin**
Appended a zero if the groupCode is not 13 digits

### CODE REVIEW

File: core.js [hunk 1]
```diff
@@ -743,6 +743,9 @@ addInputHandler('enr_glus', function (input) {
     //input = input.replace(/\W/g, '');
     state.vars.current_step = 'enr_glus';
 
+    if(input.length != 13){
+        input = '0'+ input;
+    }
```
Comment (larkinoaf) [inline core.js:748]: @shejasabin I think this if block should come after checking for "99" as otherwise we'll never let the user exit with a 99 code.

Comment (larkinoaf) [reply core.js:748]: @shejasabin maybe include this if block inside the "else"

**Review by nyabongo — CHANGES_REQUESTED** [review]
Seems okay except for the change @larkinoaf  has recommended. Line 746 should not run if the input is 99

File: core.js (hunk 1 above)
Comment (shejasabin) [reply core.js:748]: I made the changes and created a PR. I was having issues with git pushing on this one

**Pull request closed without merging**